styles/main.css

```diff
@@ -0,0 +1,173 @@
+body {
+  background-color: #deebed;
+  /* old bg color #F0F6F7*/
+}
+
+h3 {
+  opacity: 5;
+}
+
+.padded {
+  padding: 20px;
+}
+
+
```

Comment:
There's a lot of white space here, let's try to clean this file up to make it more readable.